packages/plugins/@nocobase/plugin-block-workbench/package.json

```
@@ -2,10 +2,13 @@
   "name": "@nocobase/plugin-block-workbench",
   "version": "1.0.0-alpha.17",
   "main": "dist/server/index.js",
-  "dependencies": {},
   "peerDependencies": {
     "@nocobase/client": "1.x",
     "@nocobase/server": "1.x",
     "@nocobase/test": "1.x"
+  },
+  "devDependencies": {
+    "html5-qrcode": "^2.3.8",
+    "react-router-dom": "^6.23.1"
```

```suggestion
    "react-router-dom": "6.x"
```
具体的版本号是在主仓库中确定的：https://github.com/nocobase/nocobase/blob/99e1605bd6affe4abd6702451ce2ebdf5a339c3e/package.json#L50